lms/djangoapps/instructor/enrollment.py

```diff
@@ -328,7 +328,7 @@ def _fire_score_changed_for_block(course_id, student, block, module_state_key):
         sender=None,
         points_possible=points_possible,
         points_earned=points_earned,
-        user=student,
+        user_id=student.id,
```

Comment:
I searched for all occurrences of `SCORE_CHANGED`, and manually traced each of them to ensure they could handle the new paradigm.